pkg/config/nodetreemodel/config.go

```diff
@@ -131,11 +143,13 @@ func (c *ntmConfig) OnUpdate(callback model.NotificationReceiver) {
 	c.notificationReceivers = append(c.notificationReceivers, callback)
 }
 
-func (c *ntmConfig) setDefault(key string, value interface{}) {
+func (c *ntmConfig) addToSchema(key string, source model.Source) {
 	parts := splitKey(key)
 	// TODO: Ensure that for default tree, setting nil to a node will not override
```

Comment:
I think this comment is unnecessary, it's about the default tree but this method is adding to the schema.